test/ANNOTATIONS.yaml

```diff
@@ -924,6 +927,10 @@ prk-stencil.ml-perf:
     - Update PR 5296 (#6722)
   12/02/17:
     - Stop doing a task-yield while forking remote tasks under ugni (#7915)
+  03/16/18:
+    - QSBR (#8182)
+  03/18/18:
+    - QSBR reverted (#8841)
```

Comment:
I would just apply these as global 16-node annotations since it impacted other prk variations, MG, remote executeOn perf (and some other graphs that we don't look at that often)